packages/ui/src/primitives/tailwind/Label/index.tsx

```diff
@@ -34,7 +34,7 @@ const Label = ({ className, children }: LabelProps) => {
   return (
     <label
       className={twMerge(
-        'text-theme-secondary inline-block text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70',
+        'text-theme-secondary inline-block text-sm font-medium capitalize leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70',
```

Comment:
label should not have any case by default. we should remove `capitalize`
If we need a label with capitalize, we should explicitly add make a `CapitalizedLabel` component